code/modules/mob/living/emote.dm

```diff
@@ -496,7 +497,7 @@
 			sound_to_play = H.dna.species.female_scream_sound
 		else
 			sound_to_play = H.dna.species.male_scream_sound
-	if(issilicon(user))
+	if(issilicon(user) || isipc(user))

```

Comment:
The above checks link specifically to the species scream sounds, just put the right file on `datum/species/ipc`, not here.